common/terms.html

```diff
@@ -136,7 +136,7 @@
     </dd>
     <dt><dfn data-lt="owned element|owned|owned element's|owned elements">Owned Element</dfn></dt>
     <dd>
-      <p>An 'owned element' is any <abbr title="Document Object Model">DOM</abbr> descendant of the <a>element</a>, any element specified as a child via <pref>aria-owns</pref>, or any <abbr title="Document Object Model">DOM</abbr> descendant of the owned child.</p>
+      <p>An 'owned element' is any <abbr title="Document Object Model">DOM</abbr> descendant of the <a>element</a>, any element specified as a child via <pref>aria-owns</pref>, or any <abbr title="Document Object Model">DOM</abbr> descendant of the owned child. An 'owned element' can require specific <a href="#scope">required context roles</a> in relation to their <a class="termref" href="#dfn-owning">owning elements</a>.</p>
```

Comment:
Suggestions:
- change 'require' to 'have' because 'require' is implied by 'required context roles'
- make singular/plural consistent in new sentence (the suggestion below changes to "all plural", but if preferred could change to "all singular", i.e. "An 'owned element' can have a specific required context role in relation to its owning element.")

```suggestion
      <p>An 'owned element' is any <abbr title="Document Object Model">DOM</abbr> descendant of the <a>element</a>, any element specified as a child via <pref>aria-owns</pref>, or any <abbr title="Document Object Model">DOM</abbr> descendant of the owned child. 'Owned elements' can have specific <a href="#scope">required context roles</a> in relation to their <a class="termref" href="#dfn-owning">owning elements</a>.</p>
```

Reply:
thanks!